Sources/Public/Internal/SignalContext.swift

```diff
@@ -2,7 +2,10 @@ import Foundation
 
 protocol SignalContext: CustomStringConvertible {
     var allAttributes: Attributes { get }
+    // We use attachment paths as strings in our report send functions
+    var attachmentPathsArray: [ String ] { get }
     var attributes: Attributes { get set }
+    var attachments: Attachments { get set }
```

Comment:
` our report send functions` <- report send function.. responsible for sending crashes? 
Can we also add description to `attachments` variable to avoid confussion later?

Reply:
I will clean this up, thank you!